ui/pages/multichain-accounts/account-list/account-list.tsx

```diff
@@ -49,6 +50,10 @@ export const AccountList = () => {
   const formattedAccountGroupBalancesByWallet = useAllWalletAccountsBalances();
   const [searchPattern, setSearchPattern] = useState<string>('');
 
+  // Update balances for all accounts when component mounts
+  // This ensures all account balances are visible without requiring user interaction
+  useAssetsUpdateAllAccountBalances();
```

Comment:
Nit - wondering if we can postfix/rename this to `useAssetsUpdateAllAccountBalancesEffect`. Since the key thing here is that this runs a side-effect/`useEffect` to update all account balances.